app/models/features.rb

```diff
@@ -1,7 +1,7 @@
 # Co-erces an HStore object of feature flags into a ruby model
 class Features
   include Virtus.model
-  attribute :check_in, Boolean, default: false
+  attribute :check_in, Boolean, default: true
```

Comment:
Ohhhh I read the issue and I thought it meant it wanted it turned on, but I gotcha I'll turn it back off